src/preview/src/App.css

```diff
@@ -1,43 +1,18 @@
-.App {
-  /*text-align: center;*/
-  height: 100%;
-  display: flex;
-  flex-wrap: wrap;
+html {
+  scrollbar-width: thin;
+  scrollbar-color: #fff #121212;
 
-  color: #fff;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+  &::-webkit-scrollbar {
+    width: 1rem;
   }
-}
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+  &::-webkit-scrollbar-track {
+    background-color: #121212;
   }
-  to {
-    transform: rotate(360deg);
+
+  &::-webkit-scrollbar-thumb {
+    background-color: #fff;
+    border-radius: 1rem;
+    border: 3px solid #121212;
   }
 }
```

Comment:
I see, that .App related stuff is not used anymore right? @mm-vasyl Good notice

Reply:
Yup, it had some default unused react styles